ext/liquid_c/tokenizer.c

```diff
@@ -81,7 +81,7 @@ void tokenizer_next(tokenizer_t *tokenizer, token_t *token)
           cursor++;
           token->rstrip = 1;
         }
-        if (cursor - tokenizer->cursor > 2 + token->rstrip) {
+        if (cursor - tokenizer->cursor > (intptr_t)(2 + token->rstrip)) {
```

Comment:
I agree that `ptrdiff_t` is correct here, if we're fixing this specifically may as well do it right :)